config/prow/external-secrets/public_cert.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: ppc64le-cloud-tls
```

Comment:
what's the namespace? do you want to install it in a default namespace or want to create a different namespace and host there?

Reply:
Forgot to mention, we need to apply this in prow namespace as the secret `ppc64le-cloud` with certificate is expected to be there in prow ns

Reply:
@mkumatag This secret `ppc64le-cloud` is needed in all the below namespaces:
`prow`, `grafana-dashboard`, `jenkins`, `perfdash`.

I also see the secret in a namespace by the name `jenkins-powervs`. Not sure what is the purpose of this NS.

I have deployed the External Secrets Operator in `prow` namespace. @mkumatag is it fine there or shall I move to default or a separate namespace for ESO?

Reply:
It will be good if we can deploy the external secret operator into a different namespace

@sudeeshjohn anyone is using jenkins-powervs? if not, will just delete that Jenkins server

Reply:
@mkumatag Shall we have a namespace by the name `external-secrets` and have the operator deployed there?

Reply:
@mkumatag I moved the ESO to `external-secrets` namespace and re-deployed secretstore, cert in the other four namespaces where required.
```
[root@toad1 test-external-secrets]# kubectl get pods -n external-secrets
NAME                                                READY   STATUS    RESTARTS   AGE
external-secrets-7f494f6b64-fkbqw                   1/1     Running   0          17m
external-secrets-cert-controller-76568899f9-qmwr6   1/1     Running   0          17m
external-secrets-webhook-5cd547877d-lf65m           1/1     Running   0          17m
[root@toad1 test-external-secrets]#
```
Please take a look.